src/map/status.cpp

```diff
@@ -8661,6 +8658,7 @@ static int status_get_sc_interval(enum sc_type type)
 		case SC_MAGICMUSHROOM:
 			return 4000;
 		case SC_STONE:
+		case SC_STONEWAIT:
```

Comment:
Do we need this? Stonewait doesn't deal damage.

Reply:
I think this can be removed now, just like you said.
This was used before to switch from STONEWAIT to STONE after 5 seconds.

Reply:
I put it here because I do a lookup during sc_start, but we could tell it to just look up the interval for SC_STONE.